ArchiSteamFarm/NLog/ArchiLogger.cs

```diff
@@ -151,6 +152,20 @@ public sealed class ArchiLogger {
 			Logger.Log(logEventInfo);
 		}
 
+		internal void LogInvite(ulong steamID, EAccountType accountType,  [CallerMemberName] string? previousMethodName = null) {
```

Comment:
`EAccountType` doesn't make any sense when you pass `steamID`, `steamID` already includes in itself information about account. Either remove `EAccountType`, construct `new SteamID()` from it and extract it from there, or change `ulong steamID` to `uint accountID` (and you probably want the first).

Reply:
And even better, just pass `SteamID` struct right away, no need to construct it from `ulong` then, if we want to use it anyway.